src/screens/staking/lib/staking_sdk/core.ts

```diff
@@ -69,8 +80,12 @@ export const keplrNetworks = new Set(
     StakingNetworkId.CosmosHub,
     StakingNetworkId.CosmosHubTestnet,
     StakingNetworkId.DyDx,
+    StakingNetworkId.Kava,
+    StakingNetworkId.KavaTestnet,
     StakingNetworkId.Osmosis,
     StakingNetworkId.Regen,
+    StakingNetworkId.Stargaze,
+    StakingNetworkId.StargazeTestnet,
```

Comment:
Can't comment below but if I remember correctly, had to add both testnets to `keplrNonNativeChains`

Reply:
Yup, related to the missing push 🙈